tests/cypress/support/commands.js

```diff
@@ -83,6 +83,55 @@ Cypress.Commands.add('deletingRegisteredUsers', (accountToDelete) => {
     });
 });
 
+Cypress.Commands.add('changeUserActiveStatus', (authKey, accountsToChangeActiveStatus, isActive) => {
+    let status;
+    cy.request({
+        url: '/api/v1/users?page_size=all',
+        headers: {
+            Authorization: `Token ${authKey}`,
+        },
+    }).then((response) => {
+        const responceResult = response['body']['results'];
+        for (const user of responceResult) {
```

Comment:
what do you think about this code?

```js
responseResult.filter((user) => accountsToChangeActiveStatus.includes(user['username']))
.forEach(() => {
    cy.request({
        method: 'PATCH',
        url: `/api/v1/users/${userId}`,
        headers: {
              Authorization: `Token ${authKey}`,
        },
        body: {
               is_active: isActive,
         }
     });
})

```

Reply:
Unfortunately, in this case I get error ```responseResult.filter is not a function```. But I tried to simplify this code somewhat.